src/main/java/interface_adapter/main_menu/MenuController.java

```diff
@@ -8,9 +9,12 @@
  */
 public class MenuController {
     private final MenuInputBoundary menuUseCaseInteractor;
+    private final LeaderboardInputBoundary leaderboardInteractor;
 
-    public MenuController(MenuInputBoundary menuUseCaseInteractor) {
+    public MenuController(MenuInputBoundary menuUseCaseInteractor,
+                          LeaderboardInputBoundary leaderboardInteractor) {
```

Comment:
going back to the appbuilder comment; should we be instantiating leaderboard interactors directly?